src/pages/Editor.vue

```diff
@@ -1156,6 +1186,15 @@ export default {
     ]),
     ...mapActions("generic", ["showSharePlioDialog", "showEmbedPlioDialog"]),
     ...Utilities,
+    copyPlioDraftLink() {
+      // copies the link to the clipboard
+      var success = Utilities.copyToClipboard(
+        this.getPlioDraftLink(this.plioId, this.org)
+      );
```

Comment:
Because you've spread(`...`) `Utilities` object into the methods object of this component, you can directly use those methods using `this.`

So this can be `this.copyToClipboard`
I'm mentioning this because in some components you've used utilities methods directly, using `this.`

Personally, I don't like to spread external js files into the methods object of a component and use it like `Utilities.SOME_FUNCTION` because that way it's super clear but we can use whatever but let's keep it consistent